compiler/rustc_error_messages/locales/en-US/lint.ftl

```diff
@@ -350,6 +350,8 @@ lint_builtin_mutable_transmutes =
 
 lint_builtin_unstable_features = unstable feature
 
+lint_ungated_async_fn_track_caller = `#[track_caller]` on async functions is a no-op, unless the `closure_track_caller` feature is enabled
```

Comment:
I wonder if this is better split into a primary message "... is a no-op" and a label that we only show if we're on nightly "... enable the `closure_track_caller` feature to ...".

What do you think?

Reply:
yeah that might be better, maybe something like

```
lint_ungated_async_fn_track_caller = `#[track_caller]` on async functions is a no-op
    .suggestion = enable the `closure_track_caller` feature
```
🤔
i'll look into how to only include the second part on nightly

Reply:
will add in next commit